packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -359,13 +364,21 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
 
   void _handleMouseDragEnd(DragEndDetails details) {
     _finalizeSelection();
+    if (_lastSelectedContent != _selectable?.getSelectedContent()?.plainText) {
```

Comment:
Can you refactor this into a method? it seems to be duplicated in a bunch of places

Reply:
Done :+1: